sdks/java/testing/test-utils/build.gradle

```diff
@@ -39,3 +39,14 @@ dependencies {
   testCompile library.java.hamcrest_library
   testRuntimeOnly project(path: ":runners:direct-java", configuration: "shadowTest")
 }
+
+task verifyJavaVersion(type: Test) {
```

Comment:
It is best to generalize this method since we have different options like:
Java8 Beam, Java 8 Test, Java 8 runtime
Java8 Beam, Java 11 Test, Java 11 runtime
eventually we will add
Java11 Beam, Java 8 Test, Java 8 runtime
Java11 Beam, Java 11 Test, Java 11 runtime